Makefile

```diff
@@ -102,8 +102,9 @@ shell: runcimage
 		-v $(CURDIR):/go/src/$(PROJECT) \
 		$(RUNC_IMAGE) bash
 
-install:
+install: runc recvtty
 	install -D -m0755 runc $(DESTDIR)$(BINDIR)/runc
+	install -D -m0755 contrib/cmd/recvtty/recvtty $(DESTDIR)$(BINDIR)/recvtty
```

Comment:
Not sure if we should install `recvtty`. It is not meant to be a standalone utility, but rather an example of how to write something like that.